README.es-ES.md

```
@@ -0,0 +1,308 @@
+
+### Gestión de Índices
+
+```
+"¿Qué índices existen en la colección de usuarios?"
+"Crea un índice en la colección de productos para el campo 'category'"
+"Lista todos los índices de todas las colecciones"
+```
+
+### Operaciones de Documentos
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)